modules/op-using-custom-pipeline-template-for-git-import.adoc

```diff
@@ -0,0 +1,95 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc
+
+// include::modules/common-attributes.adoc[]
+// include::modules/pipelines-document-attributes.adoc[]
+
+[id='using-custom-pipeline-template-for-git-import']
+= Using a custom pipeline template for creating and deploying an application from a Git repository
+
+:context: using-custom-pipeline-template-for-git-import
+
+toc::[]
+
+As a cluster administrator, to create and deploy an application from a Git repository, you can use custom pipeline templates that override the default pipeline templates provided by {pipelines-title} 1.5 and later.
+
+[NOTE]
+====
+This feature is unavailable in {pipelines-title} 1.4 and earlier versions.
+====
+
+.Prerequisites
+
+Ensure that the {pipelines-title} 1.5 or later is installed and available in all namespaces.
+
+.Procedure
+
+. Log in to the {product-title} web console as a cluster administrator.
+. In the *Administrator* perspective, using the left navigation panel, go to *Pipelines*.
+.. From the *Project* drop-down, select the *openshift* project. This ensures that the subsequent steps are performed in the `openshift` namespace.
+.. From the list of available pipelines, select a pipeline that is appropriate for building and deploying your application. For example, if your application requires a `node.js` runtime environment, select the *s2i-nodejs* pipeline.
++
+[NOTE]
+====
+Do not edit the default pipeline template. It may become incompatible with the UI and the back-end.
+====
++
+.. Under the *YAML* tab of the selected pipeline, click *Download* and save the YAML file to your local machine. If your custom configuration file fails, you can use this copy to restore a working configuration.
+. Using the left navigation panel, go to **Operators** -> **Installed Operators**.
+.. Click *{pipelines-title}* -> *Tekton Configuration* tab -> *config* -> *YAML* tab.
+.. To disable (delete) the default pipeline templates in the `openshift` namespace, set the `pipelineTemplates` parameter to `false`  in the `TektonConfig` custom resource YAML, and save it.
++
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+profile: all
+targetNamespace: openshift-pipelines
+addon:
+  params:
+  - name: clusterTasks
+    value: "true"
+  - name: pipelineTemplates
+    value: "false"
+...
+----
++
+[NOTE]
+====
+If you manually delete the default pipeline templates, the operator will restore the defaults during an upgrade.
+====
++
+[WARNING]
+====
+As a cluster admin, you can disable installation of the default pipeline templates in the operator configuration. However, such a configuration will disable (delete) all default pipeline templates, and not only the one you want to customize.
+====
++
+. Using the left navigation panel, go to *Pipelines*.
```

Comment:
Same as step 3, suggest this for step 4:
Create a custom pipeline template:
this step can becom the first sub-step under Step 4: . Use the left navigation panel to go to the *Pipelines* section.